### PR TITLE
feature/update_baseline_markdown

### PR DESCRIPTION
**Pull request by dawit-simeon**

### CODE REVIEW

File: baseline.md [hunk 1]
```diff
@@ -1,68 +1,3 @@
-# The Simeon Baseline: A comprehensive set of best practice Microsoft 365 configurations.
```
Comment (jeff-simeon) [inline baseline.md:1]: the top part here was removed because the Build-BaselineMarkdown script was not run from the docs repo - the working directory when running the script should be the docs repo. Revert these changes before running again (because it won't put it back now that it's been removed)

File: baseline.md [hunk 2]
```diff
@@ -655,14 +563,14 @@ Not yet a client of Simeon? [Get started here](https://www.simeoncloud.com/).
 
 ###### hidden-header
 
-### Xbox Console Companion
+### windowsMicrosoftEdgeApp - Microsoft Edge for Windows 10
```
Comment (jeff-simeon) [inline baseline.md:566]: these are expected since we changed the naming convention for MobileApps

File: baseline.md [hunk 3]
```diff
@@ -1158,9 +1066,9 @@ Not yet a client of Simeon? [Get started here](https://www.simeoncloud.com/).
 
 ###### hidden-header
 
-### Baseline - Self Deploying Profile
+### Baseline%3A Self Deploying Profile
```
Comment (jeff-simeon) [inline baseline.md:1069]: the Build-BaselineMarkdown script needs to be enhanced to check for a $name property and use it if present (rather than always using the file name)

**Review by jeff-simeon — COMMENTED** [review]
see comments

**Review by jeff-simeon — APPROVED** [review]

**Review by rdorta-simeon — DISMISSED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Review by rdorta-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]
